src/GeoJSON.js

```diff
@@ -0,0 +1,83 @@
+import {Transform} from 'vega-dataflow';
+import {error, inherits, isArray} from 'vega-util';
+
+/**
+ * Convert an array of longitude/latitude points or the collection of the specified field
+ * into GeoJSON for use in a projection's fit argument.
+ * @constructor
+ * @param {object} params - The parameters for this operator.
+ * @param {Array<function(object): *>} params.fields - A two-element array of
+ *   field accessors for the longitude and latitude values.
+ */
+export default function GeoJSON(params) {
+  Transform.call(this, null, params);
+}
+
+export function collectGeoJSON(features) {
+  return !isArray(features) ? features
+    : features.length > 1 ? {type: 'FeatureCollection', features: features}
+    : features[0];
+}
+
+GeoJSON.Definition = {
+  "type": "GeoJSON",
+  "metadata": {},
+  "params": [
+    { "name": "fields", "type": "field", "array": true, "length": 2 },
+    { "name": "geojson", "type": "field" },
+  ]
+};
+
+var prototype = inherits(GeoJSON, Transform);
+
+prototype.transform = function(_, pulse) {
+  var fields = _.fields,
+      lon = fields[0],
+      lat = fields[1],
+      geojson = _.geojson,
+      flag = pulse.ADD,
+      features = [],
```

Comment:
To handle streaming data, you'll likely want to make `features` available as a stored property of the transform object (e.g., `this._features`). If the `mod` flag is true and you re-visit all tuples, then reset this array to `[]`. Otherwise use the existing version. In the current implementation, for streaming updates with `mod == false`, only the data in the `ADD` set is included in this operator's `value`; any previous values are getting dropped!